workspace/viewer/PdfJsViewer.ts

```diff
@@ -464,18 +470,20 @@ export class PdfJsViewer {
       canvas.removeClass("shadow");
     }
     const canvasOffset = canvas.position();
-    const textLayer = $(".textLayer", this.jqRoot);
+    const textLayer = parent.find(".textLayer");
     textLayer.css({
       top : canvasOffset.top,
       left : canvasOffset.left
     });
   }
 
   resetCanvas() {
-    if (this.pdfPageView) {
-      this.pdfPageView.destroy();
+    if (this.pdfPageView.length != 0) {
+      for(var i = 0; i != this.pdfPageView.length; i++){
```

Comment:
[Make loops great again](https://basarat.gitbooks.io/typescript/docs/for...of.html). `for (let page of this.pdfPageView)`

Reply:
Done.